src/NLog/LayoutRenderers/Wrappers/ObjectPathRendererWrapper.cs

```diff
@@ -0,0 +1,153 @@
+// 
+// All rights reserved.
+// 
+// Redistribution and use in source and binary forms, with or without 
+// modification, are permitted provided that the following conditions 
+// are met:
+// 
+// * Redistributions of source code must retain the above copyright notice, 
+//   this list of conditions and the following disclaimer. 
+// 
+// * Redistributions in binary form must reproduce the above copyright notice,
+//   this list of conditions and the following disclaimer in the documentation
+//   and/or other materials provided with the distribution. 
+// 
+// * Neither the name of Jaroslaw Kowalski nor the names of its 
+//   contributors may be used to endorse or promote products derived from this
+//   software without specific prior written permission. 
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE 
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE 
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE 
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR 
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS 
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN 
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) 
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF 
+// THE POSSIBILITY OF SUCH DAMAGE.
+// 
+
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using NLog.Config;
+using NLog.Internal;
+using NLog.Layouts;
+
+namespace NLog.LayoutRenderers.Wrappers
+{
+    /// <summary>
+    /// Render a single property of a object
+    /// </summary>
+    [LayoutRenderer("Object-Path")]
+    [AmbientProperty(nameof(ObjectPath))]
+    [ThreadAgnostic]
+    [ThreadSafe]
+    public class ObjectPathRendererWrapper : WrapperLayoutRendererBase, IRawValue
+    {
+        private readonly ObjectPropertyHelper _objectPropertyHelper = new ObjectPropertyHelper();
+
+        /// <inheritdoc />
+        public ObjectPathRendererWrapper()
+        {
+            InnerChanged += ObjectPathRendererWrapper_InnerChanged;
+            UpdateInner();
+        }
+
+        private void ObjectPathRendererWrapper_InnerChanged(object sender, EventArgs e)
+        {
+            UpdateInner();
+        }
+
+        private void UpdateInner()
+        {
+            _innerAsSimple = Inner as SimpleLayout;
+        }
+
+
+        private SimpleLayout _innerAsSimple;
+
+        /// <summary>
+        /// Gets or sets the object-property-navigation-path for lookup of nested property
+        ///
+        /// Shortcut for <see cref="ObjectPath"/>
+        /// </summary>
+        public string Path
+        {
+            get => ObjectPath;
```

Comment:
@snakefoot not keen on this, but otherwise you need this:

```
${object-path:${exception}:objectpath=ParamName}
```

PS dashes won't work in ambient properties

Reply:
Why is this not possible `${exception:objectpath=ParamName}` ?

Reply:
yes that works!

This works now:

```
${object-path:${exception}:path=ParamName}
${exception:objectpath=ParamName}
```

But I don't prefer: (therefor the delegated property)

```
${object-path:${exception}:objectpath=ParamName}
```

and also this isn't possible: (dashed ambient property)

```
${exception:object-path=ParamName}
```